config/ignition_edifice_debian_buster.yaml

```diff
@@ -2,120 +2,148 @@ name: ignition_edifice_debian_buster
  Package (= libignition-common4-dev) |\
  Package (= libignition-common4-events) |\
+ Package (= libignition-common4-events-dbgsym) |\
  Package (= libignition-common4-events-dev) |\
  Package (= libignition-common4-graphics) |\
+ Package (= libignition-common4-graphics-dbgsym) |\
  Package (= libignition-common4-graphics-dev) |\
  Package (= libignition-common4-profiler) |\
- Package (= libignition-common4-profiler-dev)
-), $Version (% 4.4.0-*) |\
+ Package (= libignition-common4-profiler-dbgsym) |\
+ Package (= libignition-common4-profiler-dev) \
+), $Version (% 4.5.0-1*) |\
 (Package (= ignition-fuel-tools6) |\
  Package (= libignition-fuel-tools6) |\
- Package (= libignition-fuel-tools6-dev)
-), $Version (% 6.1.0-*) |\
+ Package (= libignition-fuel-tools6-dbgsym) |\
+ Package (= libignition-fuel-tools6-dev) \
+), $Version (% 6.2.0-1*) |\
 (Package (= ignition-gazebo5) |\
  Package (= libignition-gazebo5) |\
+ Package (= libignition-gazebo5-dbg) |\
  Package (= libignition-gazebo5-dev) |\
- Package (= libignition-gazebo5-plugins)
-), $Version (% 5.3.0-*) |\
+ Package (= libignition-gazebo5-plugins) \
+), $Version (% 5.2.0-1*) |\
 (Package (= ignition-gui5) |\
  Package (= libignition-gui5) |\
- Package (= libignition-gui5-dev)
-), $Version (% 5.3.0-*) |\
+ Package (= libignition-gui5-dbgsym) |\
+ Package (= libignition-gui5-dev) \
+), $Version (% 5.5.0-1*) |\
 (Package (= ignition-launch4) |\
  Package (= libignition-launch4) |\
- Package (= libignition-launch4-dev)
-), $Version (% 4.1.0-*) |\
+ Package (= libignition-launch4-dbgsym) |\
+ Package (= libignition-launch4-dev) \
+), $Version (% 4.1.0-1*) |\
 (Package (= ignition-msgs7) |\
  Package (= libignition-msgs7) |\
- Package (= libignition-msgs7-dev) |\
- Package (= libignition-msgs7-dbg)
-), $Version (% 7.2.0-*) |\
+ Package (= libignition-msgs7-dbgsym) |\
+ Package (= libignition-msgs7-dev) \
+), $Version (% 7.3.0-1*) |\
 (Package (= ignition-physics4) |\
  Package (= libignition-physics4) |\
- Package (= libignition-physics4-bullet-dev) |\
  Package (= libignition-physics4-bullet) |\
+ Package (= libignition-physics4-bullet-dbgsym) |\
+ Package (= libignition-physics4-bullet-dev) |\
  Package (= libignition-physics4-core-dev) |\
  Package (= libignition-physics4-dartsim) |\
+ Package (= libignition-physics4-dartsim-dbgsym) |\
  Package (= libignition-physics4-dartsim-dev) |\
+ Package (= libignition-physics4-dbgsym) |\
  Package (= libignition-physics4-dev) |\
  Package (= libignition-physics4-heightmap-dev) |\
  Package (= libignition-physics4-mesh-dev) |\
  Package (= libignition-physics4-sdf-dev) |\
  Package (= libignition-physics4-tpe) |\
+ Package (= libignition-physics4-tpe-dbgsym) |\
  Package (= libignition-physics4-tpe-dev) |\
  Package (= libignition-physics4-tpelib) |\
- Package (= libignition-physics4-tpelib-dev)
-), $Version (% 4.3.0-*) |\
+ Package (= libignition-physics4-tpelib-dbgsym) |\
+ Package (= libignition-physics4-tpelib-dev) \
+), $Version (% 4.2.0-2*) |\
```

Comment:
Previously configured: 4.3.0
Actually available in the repository: 4.2.0

Currently imported in ROS: 4.1.0